packages/search/src/content/index.ts

```diff
@@ -89,8 +102,15 @@ export class ContentSearchService
  * ```
  */
 export function searchContent(
-  request: IContentSearchRequest = { filter: {}, options: {} }
+  request: IContentSearchRequest = { filter: {}, options: {} },
+  hubRequestOptions: IHubRequestOptions
```

Comment:
Now you _can_ pass in your own `IHubRequestOptions`, but if you don't it will do its best to build one for you.

Reply:
I think this is reasonable